common/src/main/resources/tachyon-default.properties

```diff
@@ -39,6 +39,10 @@ tachyon.max.columns=1000
 tachyon.test.mode=false
 tachyon.max.table.metadata.byte=5MB
 tachyon.host.resolution.timeout.ms=5000
+#Path where executors will load their dependencies. must be path in S3, HDFS etc..
+tachyon.executor.dependency.path=https://s3.amazonaws.com/bucketname/foldername/
```

Comment:
maybe rename these properties prefixed with `tachyon.integration.mesos` to be more specific on their goal. Otherwise, I thought their functions are targeted at the global Tachyon scope.